rsocket-transport-netty/src/main/java/io/rsocket/transport/netty/TcpDuplexConnection.java

```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2015-2018 the original author or authors.
+ * Copyright (c) 2011-2018 Pivotal Software Inc, All Rights Reserved.
```

Comment:
Doesn't match the contributing or license recommendations.  What is the backstory behind this change?

Reply:
copy and paste from another branch - should be the original author or authors.

Reply:
my bad sorry i'll update my intelliJ template for the next time